OMCompiler/Compiler/SimCode/SimCodeMain.mo

```diff
@@ -1082,7 +1082,7 @@ algorithm
           timeTemplates := System.realtimeTock(ClockIndexes.RT_CLOCK_TEMPLATES);
         end if;
       then (false, {}, "");
-  end matchcontinue;
+  end match;
```

Comment:
(Y)

Reply:
I knew you would like that! :D

I will try to do that whenever i see a matchcontinue, but as i said. Sometimes matchcontinue results in weird behaviour and i don't know how and if i should reproduce it by replacing it with match. In this case it was straight forward.